XXX
"By default pthread mutexes aren't recursive, this would be a problem if exiv2
actually wants recursive mutexes.  With no static initializer for this type of
mutex, I'll admit that it's a bit cumbersome to use." -- jca@

Index: src/actions.cpp
--- src/actions.cpp.orig
+++ src/actions.cpp
@@ -1923,11 +1923,7 @@ namespace {
     static pthread_mutex_t cs =  PTHREAD_MUTEX_INITIALIZER;
   #endif
  #else
-  #if defined(PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP)
-    static pthread_mutex_t cs =  PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP;
-   #else
     static pthread_mutex_t cs =  PTHREAD_MUTEX_INITIALIZER;
-  #endif
  #endif
 #endif
 
